packages/beacon-node/test/spec/utils/specTestIterator.ts

```diff
@@ -59,7 +59,7 @@ const coveredTestRunners = [
 // ],
 // ```
 export const defaultSkipOpts: SkipOpts = {
-  skippedForks: ["eip7732", "eip7805", "gloas"],
+  skippedForks: ["eip7732", "eip7805"],
```

Comment:
we should probably remove `eip7732` as those tests don't exist anymore
```suggestion
  skippedForks: ["eip7805"],
```